apps/myaccount/src/components/change-password/change-password.tsx

```diff
@@ -80,6 +81,67 @@ export const ChangePassword: FunctionComponent<ChangePasswordProps> = (props: Ch
 
     const endUserSession = useEndUserSession();
 
+    interface iconProps {
+        class : string;
+        color : SemanticCOLORS;
+    }
+
+    const getIconProps = (id : string) : iconProps =>{
+                
+        const iconProps = {
+            class : "circle",
+            color : "grey" as SemanticCOLORS
+        };
+
+        const lowerCaseLetters = /[a-z]/g;
+        const upperCaseLetters = /[A-Z]/g;
+        const numbers = /[0-9]/g;
+        const chars = /[!@#$%^&*~]/g;
+
+        if ( id === "password-validation-length" ){
+            if (password === "") {
```

Comment:
Use a constant

const EMPTY_STRING = "";

```suggestion
            if (password === EMPTY_STRING) {
```